package.json

```diff
@@ -102,13 +104,14 @@
         "tag-pattern": ".+"
     },
     "scripts": {
-        "clean": "rm -rf ./dist && mkdir dist",
+        "clean": "rimraf ./dist && mkdir dist",
         "compile": "npm run clean && tsc -p .",
         "prepublishOnly": "npm run build",
-        "build": "npm run compile && cp -r src/lib/framework ./dist/lib/framework",
+        "build": "npm run compile && cpx \"src/lib/framework/**/*\" ./dist/lib/framework",
         "lint": "eslint \"src/**/*.ts\"",
         "cli": "npm run compile && npm unlink && npm link",
-        "test": "npm run build && nyc mocha src/**/*.spec.ts --full-trace --bail",
+        "test": "nyc mocha",
+        "test:nocover": "mocha",
```

Comment:
When running unit tests in vscode, it's faster to run the tests without coverage, so the `"test"` task in vscode is now bound to this non-coverage npm script.